lib/cs_guide/resources/venue.ex

```diff
@@ -12,8 +12,16 @@ defmodule CsGuide.Resources.Venue do
     field(:postcode, :string)
     field(:phone_number, :string)
     field(:cs_score, :float, default: 0.0)
+    field(:num_cocktails, :integer)
     field(:entry_id, :string)
     field(:deleted, :boolean, default: false)
+    field(:description, :string)
+    field(:website, :string)
+    field(:address_line_1, :string)
```

Comment:
> `:address` is discussed in: https://github.com/dwyl/fields/issues/21
> we _must_ create and use the _modular/reusable_ field 
> **so that** we don't have to keep re-writing address validation and sanitisation code `forEach` new project.